multiversx_sdk_cli/cli_shared.py

```diff
@@ -76,6 +76,12 @@ def add_tx_args(args: List[str], sub: Any, with_nonce: bool = True, with_receive
 
     sub.add_argument("--chain", default=config.get_chain_id(), help="the chain identifier (default: %(default)s)")
     sub.add_argument("--version", type=int, default=config.get_tx_version(), help="the transaction version (default: %(default)s)")
+
+    if with_guardian:
+        sub.add_argument("--guardian-address", type=str, help="the address of the guradian")
+        sub.add_argument("--guardian-service-url", type=str, help="the url of the guardian service")
+        sub.add_argument("--guardian-code", type=str, help="the 2fa code for the guardian")
```

Comment:
"Code" is an ambiguous term. Let's check with Adi what name should we use.

Reply:
The code parameter is now called `--guardian-2fa-code`